openapi/tests/input_output/input.py

```diff
@@ -22,6 +22,22 @@ class CogniteListUpdate:
     pass
 
 
+class CognitePropertyClassUtil:
```

Comment:
why does this need mocking instead of importing?

Reply:
I don't want to import anything from the cognite.client here due to the added dependency. Mocked instead of just declared as it is called at collection stage by generated code.